app/models/setting.rb

```diff
@@ -56,12 +56,8 @@ def validate(record)
   after_find :readonly_when_overridden
   default_scope -> { order(:name) }
 
-  # Filer out settings from disabled plugins and ones releated to taxonomies when required
+  # Filer out settings from disabled plugins
   scope :disabled_plugins, -> { where(:category => self.descendants.map(&:to_s)) unless Rails.env.development? }
-  scope :default_organization, -> { where('name not in (?)', 'default_organization') unless Taxonomy.enabled_taxonomies.include? 'organizations' }
```

Comment:
You have removed these scopes, it can cause breakage in the code. Let's deprecate them instead:
``` ruby
scope :default_organization, -> { Foreman::Deprecation.deprecation_warning('1.23', 'this scope no longer filter anything since taxonomies cannot be disabled in 1.21.'); where(nil) }
```

Reply:
I don't think these scopes are used anywhere except below, but deprecated just in case.